### Data and Project Management 

The purpose of this notebook is to download, assess, and explore the data that will be worked with in this Neurocognitive psychology project. The dataset used is the Autism Brain Imaging Data Exchange II (ABIDE II) dataset. ABIDE II was developed as a continuation to the ABIDE I dataset as part of the 1000 Functional Connectomes Project, to further encourage discovery science on the brain connectome in Autism Spectrum Disorder. This dataset consists of MRI data of patients with and without Autism, from various different sites around the world. From this dataset we will be using the anatomical data, and the CSV file with all phenotypic information for the participants.  

#### *Loading the data*

Access to the ABIDE II dataset is granted through the NITRC website, a website where Neuroimaging Tools and Resources are shared. Here we can find the phenotpyic CSV file. Before downloading the actual brain images, it is important to get a general overview of what data we are dealing with. To do this, we begin by downloading the .csv file, and reading it as a dataframe.  

In [3]:
from os import chdir

chdir('/Users/gretivan/Desktop/Neuro_data/')

In [9]:
import pandas as pd

Data_overview = pd.read_csv("Data_overview.csv")
Data_overview

Unnamed: 0  Unnamed: 0.1  SUB_ID     X  subject  SITE_ID  \
0              0             1   50002     1    50002     PITT   
1              1             2   50003     2    50003     PITT   
2              2             3   50004     3    50004     PITT   
3              3             4   50005     4    50005     PITT   
4              4             5   50006     5    50006     PITT   
...          ...           ...     ...   ...      ...      ...   
1107        1107          1108   51583  1108    51583      SBL   
1108        1108          1109   51584  1109    51584      SBL   
1109        1109          1110   51585  1110    51585      SBL   
1110        1110          1111   51606  1111    51606  MAX_MUN   
1111        1111          1112   51607  1112    51607  MAX_MUN   

               FILE_ID  DX_GROUP  DSM_IV_TR  AGE_AT_SCAN  ...  \
0          no_filename         1          1        16.77  ...   
1         Pitt_0050003         1          1        24.45  ...   
2         Pitt_0050004         1          1        19.09  ...   
3         Pitt_0050005         1          1        13.73  ...   
4         Pitt_0050006         1          1        13.37  ...   
...                ...       ...        ...          ...  ...   
1107       SBL_0051583         1          2        35.00  ...   
1108       SBL_0051584         1          2        49.00  ...   
1109       SBL_0051585         1          1        27.00  ...   
1110  MaxMun_a_0051606         1          2        29.00  ...   
1111  MaxMun_a_0051607         1          2        26.00  ...   

      qc_notes_rater_1 qc_anat_rater_2  qc_anat_notes_rater_2  \
0                  NaN              OK                    NaN   
1                  NaN              OK                    NaN   
2                  NaN              OK                    NaN   
3                  NaN              OK                    NaN   
4                  NaN              OK                    NaN   
...                ...             ...                    ...   
1107               NaN              OK                    NaN   
1108               NaN              OK                    NaN   
1109               NaN              OK                    NaN   
1110               NaN              OK                    NaN   
1111               NaN              OK                    NaN   

      qc_func_rater_2        qc_func_notes_rater_2  qc_anat_rater_3  \
0                fail       ic-parietal-cerebellum               OK   
1                  OK                          NaN               OK   
2                  OK                          NaN               OK   
3               maybe       ic-parietal-cerebellum               OK   
4               maybe           ic-parietal slight               OK   
...               ...                          ...              ...   
1107               OK  ic-cerebellum-temporal_lobe               OK   
1108            maybe                vmpfc dropout               OK   
1109            maybe  ic-cerebellum-temporal_lobe               OK   
1110            maybe                ic-cerebellum               OK   
1111            maybe                ic-cerebellum               OK   

     qc_anat_notes_rater_3 qc_func_rater_3 qc_func_notes_rater_3  SUB_IN_SMP  
0                      NaN            fail             ERROR #24           1  
1                      NaN              OK                   NaN           1  
2                      NaN              OK                   NaN           1  
3                      NaN              OK                   NaN           0  
4                      NaN              OK                   NaN           1  
...                    ...             ...                   ...         ...  
1107                   NaN              OK                   NaN           0  
1108                   NaN              OK                   NaN           0  
1109                   NaN              OK                   NaN           0  
1110    

We can see here that there are 1112 subjects in this dataset. 